java-client/src/test/java/org/opensearch/client/opensearch/json/JsonpSerializableTest.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+/*
```

Comment:
Please keep only SPDX license, this is a new code, no attribution to Elastic

Reply:
Alright , let me fix it.